lib/ansible/modules/net_tools/nsupdate.py

```diff
@@ -76,7 +76,12 @@
     value:
         description:
             - Sets the record value.
-
+    protocol:
+        description:
+            - Sets the transport protocol
```

Comment:
Perhaps we could have a few words more here, clarifying that tcp generally is the preferred and more robust choice?